packages/graphback/src/resolvers/knex/resolverImplementation.ts

```diff
@@ -22,13 +23,14 @@ export const findAllTemplate = (fieldName: string, tableName: string): string =>
 
 export const findTemplate = (fieldName: string, tableName: string): string => {
     return `${fieldName}: (root, args, context, info) => {
-      return ${knexContext}.select().from('${tableName}').where(...${argumentContext}.fields)
+      return ${knexContext}.select().from('${tableName}').where(${argumentContext}.fields)
     }`
 }
 
 export const updateTemplate = (fieldName: string, tableName: string): string => {
     return `${fieldName}: (root, args, context, info) => {
```

Comment:
What happened to our type safety claim? Does this compile with TypeScript engine (it should not)

Reply:
Haven't made them type safe. Should we use graphql-codegen and generate types?

Reply:
Yes.. Why not? Any challenge?